.github/workflows/observability-ci.yml

```
@@ -676,43 +676,43 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
-      
+
       - name: Set up kubectl
         uses: azure/setup-kubectl@v3
         with:
           version: '1.28.0'
-      
+
       - name: Azure Login
         uses: azure/login@v1
```

actionlint (1.7.10)

[error] the runner of "azure/login@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)